src/API/Site/Controllers/MerchantCenter/RequestReviewController.php

```diff
@@ -21,18 +23,19 @@
  */
 class RequestReviewController extends BaseOptionsController {
 
-
 	/**
 	 * RequestReviewController constructor.
 	 *
 	 * @param RESTServer            $server
 	 * @param Middleware            $middleware
 	 * @param RequestReviewStatuses $request_review_statuses
+	 * @param TransientsInterface   $transient
 	 */
-	public function __construct( RESTServer $server, Middleware $middleware, RequestReviewStatuses $request_review_statuses ) {
+	public function __construct( RESTServer $server, Middleware $middleware, RequestReviewStatuses $request_review_statuses, TransientsInterface $transient ) {
 		parent::__construct( $server );
 		$this->middleware              = $middleware;
 		$this->request_review_statuses = $request_review_statuses;
+		$this->transient               = $transient;
```

Comment:
This is the interface for transients, so I think it's best to name it plural `transients`. Also we should probably declare it in the class so we can make it protected (otherwise it defaults to public).